docusaurus/docs/learn/quickstarts/network/miniziti.bash

```diff
@@ -774,34 +792,51 @@ main(){
         testClusterDns "${MINIKUBE_NODE_EXTERNAL}"
     fi
 
+    if miniziti kubectl get configmaps \
```

Comment:
Should this be `kubectlWrapper` instead of `miniziti`?

Reply:
yes

Reply:
This should be resolved.